Error while checking build/AliceO2/O2/o2/macOS-arm for d99bbd9305cbc66a2eb99fab135c7c70bb123210 at 2022-09-09 14:01:
```
## sw/BUILD/O2-latest/log
 97/446 Test  #89: CCDB/test/testCcdbApi.cxx ............................................................***Failed    7.91 sec
<!doctype html><html lang="en"><head><title>HTTP Status 500 – Internal Server Error</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 500 – Internal Server Error</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Cannot upload the blob to the local file &#47;root&#47;QC&#47;60&#47;8071&#47;24e9a501-3036-11ed-8a02-c0a80209250c</p><p><b>Description</b> The server encountered an unexpected condition that prevented it from fulfilling the request.</p><hr class="line" /></body></html>[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPath/1662724462351/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:142: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check path2 != nullptr has failed [0x0 == nullptr][0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:145: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check path2 && path2->getPathString().CompareTo("HelloWorld") == 0 has failed[0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPath/1662724462560/Hello=World/
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPath/1662724462764/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:153: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check md.count("Hello") == 1 has failed [0 != 1][0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:154: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check md["Hello"] == "World" has failed [ != World][0;39;49m
<!doctype html><html lang="en"><head><title>HTTP Status 500 – Internal Server Error</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 500 – Internal Server Error</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Cannot upload the blob to the local file &#47;root&#47;QC&#47;73&#47;2908&#47;256bb04e-3036-11ed-8a02-c0a80209250c</p><p><b>Description</b> The server encountered an unexpected condition that prevented it from fulfilling the request.</p><hr class="line" /></body></html>[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/tree2/1662724463203/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:168: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check tree != nullptr has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:169: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check tree != nullptr && std::strcmp(tree->GetName(), "tree123") == 0 has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:170: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check tree != nullptr && tree->GetEntries() == 1 has failed[0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013//1662724463409/
[ERROR] Unable to find object Test/pid26013//1662724463409, Aborting
[ERROR] Local snapshot /tmp/sQzldVI5ZuGXbEWR/Test/pid26013/CCDBPath/snapshot.root not found 
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:188: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check snapshot.retrieveFromTFileAny<o2::ccdb::IdPath>(basePath + "CCDBPath", f.metadata) != nullptr has failed[0;39;49m
[ERROR] Local snapshot /tmp/sQzldVI5ZuGXbEWR/Test/pid26013/tree2/snapshot.root not found 
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:192: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check tree != nullptr has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:193: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check tree != nullptr && std::strcmp(tree->GetName(), "tree123") == 0 has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:194: [1;31;49merror: in "store_retrieve_TMemFile_templated_test": check tree != nullptr && tree->GetEntries() == 1 has failed[0;39;49m
<!doctype html><html lang="en"><head><title>HTTP Status 500 – Internal Server Error</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 500 – Internal Server Error</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Cannot upload the blob to the local file &#47;root&#47;QC&#47;44&#47;9086&#47;260c4cd5-3036-11ed-8a02-c0a80209250c</p><p><b>Description</b> The server encountered an unexpected condition that prevented it from fulfilling the request.</p><hr class="line" /></body></html>[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1000/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:237: [1;31;49merror: in "timestamptest": check path2 != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1000/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1001/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1002/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1003/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1004/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1005/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1006/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1007/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1008/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1009/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:242: [1;31;49merror: in "timestamptest": check p != nullptr has failed [0x0 == nullptr][0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/999/Hello=World/
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1010/Hello=World/
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/CCDBPathUnitTest/1662724466966/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:270: [1;31;49merror: in "retrieveTemplatedWithHeaders": check headers.count("Hello") == 1 has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:271: [1;31;49merror: in "retrieveTemplatedWithHeaders": check headers["Hello"] == "World" has failed[0;39;49m
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/Test/pid26013/th1/1662724467387/Hello=World/
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9784/0/CCDB/test/testCcdbApi.cxx:279: [1;31;49merror: in "retrieveTMemFile_test": check obj != nullptr has failed [0x0 == nullptr][0;39;49m
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9784/d99bbd9305cbc66a2eb99fab135c7c70bb123210/build_AliceO2_O2_o2_macOS-arm/pretty.html).

--- a/prodtests/full-system-test/aggregator-workflow.sh
+++ b/prodtests/full-system-test/aggregator-workflow.sh
@@ -202,7 +202,7 @@ lanesFactorize=6
 nTFs=1000
 
 if ! workflow_has_parameter CALIB_LOCAL_INTEGRATED_AGGREGATOR && [[ $CALIB_TPC_IDC == 1 ]] && [[ $AGGREGATOR_TASKS == TPCIDC_A || $AGGREGATOR_TASKS == TPCIDC_C || $AGGREGATOR_TASKS == TPCIDC_BOTH || $AGGREGATOR_TASKS == ALL ]]; then
-  add_W o2-tpc-idc-distribute "--crus ${crus} --timeframes ${nTFs} --firstTF -100 --output-lanes ${lanesFactorize} --send-precise-timestamp true --condition-tf-per-query ${nTFs}"
+  add_W o2-tpc-idc-distribute "--crus ${crus} --timeframes ${nTFs} --output-lanes ${lanesFactorize} --send-precise-timestamp true --condition-tf-per-query ${nTFs}"
   add_W o2-tpc-idc-factorize "--input-lanes ${lanesFactorize} --crus ${crus} --timeframes ${nTFs} --nthreads-grouping 8 --nthreads-IDC-factorization 8 --sendOutputFFT true --nTFsMessage 500 --enable-CCDB-output true --enablePadStatusMap true --use-precise-timestamp true" "TPCIDCGroupParam.groupPadsSectorEdges=32211"
   add_W o2-tpc-idc-ft-aggregator "--rangeIDC 200 --inputLanes ${lanesFactorize} --nFourierCoeff 40 --nthreads 8"
 fi

